packages/back-end/src/events/handlers/slack/slack-event-handler-utils.ts

```diff
@@ -0,0 +1,36 @@
+import { KnownBlock } from "@slack/web-api";
+import fetch from "node-fetch";
+import { logger } from "../../../util/logger";
+
+export type SlackMessage = {
+  text: string;
+  blocks: KnownBlock[];
+};
+
+/**
+ * Sends a Slack message.
+ * @param slackMessage
+ * @param webHookEndpoint
+ * @throws Error If the request fails
+ */
+export const sendSlackMessage = async (
+  slackMessage: SlackMessage,
+  webHookEndpoint: string
+): Promise<boolean> => {
+  try {
+    const response = await fetch(webHookEndpoint, {
+      method: "POST",
+      body: JSON.stringify(slackMessage),
+    });
```

Comment:
Yes, I think that's a good idea. I've made the following changes:

- Refactored the `cancellableFetch` to a more global `http.util` file: b4ee7ab31ecf1d09a77fe2c8d76a2a611846c2ae
- Implement it: a905937804362f93f7f1c64f1b17cbdab5c97387
- Improve Slack URL validation: 1bfecb3787361d433233187726f81a847b703b46

It's highly unlikely Slack would take a long time to respond or send a big message, but I've added those precautions to enforce they're using Slack URL's, and just in case something goes horribly wrong at Slack HQ, though it's highly unlikely.